ip/draft-ietf-detnet-ip.xml

```diff
@@ -1000,8 +1005,9 @@ DetNet                    |L2/SbN|          |L2/SbN|
             configuration, e.g., via the controller or management plane.
           </t>
           <t>
-            Information identifying a DetNet flow is ordered and
-            implementations use the first match.  This can, for example, be
+            An implementation MUST support ordering of the
+            set of information information used to identify an
+            individual DetNet flow.  This can, for example, be
```

Comment:
An implementation MUST support using a set of information to identify an individual flow to support ordering for that flow.